src/games/strategy/engine/framework/map/download/DownloadRunnable.java

```diff
@@ -34,7 +36,11 @@ public DownloadRunnable(final String urlString) {
   }
 
   public List<DownloadFileDescription> getDownloads() {
-    return downloads;
+   if(downloads == null) {
```

Comment:
Addressed in: 52392020f428e6b7e93e5d71913c8bfc4038aa6d
Really we should not even have the 'downloads' variable to begin with.